activities/intermediate/higher-order-array-functions/src/example/example.js

```diff
@@ -0,0 +1,37 @@
+// map
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map
+
+const arr = [1, 2, 3, 4, 5];
+
+// assign a function to the const mapExample that will
+// accept one argument, an array
+const mapExample = arr => arr.map(x => x * 2);
+
+console.log(mapExample(arr)); 
+// returns [ 2, 4, 6, 8, 10 ]
+
+// filter
+// https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/filter
+
+const arr1 = [1, '', true, false, null, undefined];
+
+// assign a function to the const filterExample that will
+// accept one argument, an array
+const filterExample = arr => arr.filter(x => !!x);
```

Comment:
Double bang isn't really required here, I think it'd throw some people off since it's not often seen in my experience.